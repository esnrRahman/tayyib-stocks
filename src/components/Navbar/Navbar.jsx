import React from 'react';
import { Menu } from 'semantic-ui-react';

const Navbar = () => (
  <Menu fluid widths={5}>
    <Menu.Item name="Home" />
    <Menu.Item name="About Us" />
    <Menu.Item name="Shariah Compliance" />
    <Menu.Item name="Helpful Websites" />
    <Menu.Item name="Contact Us" />
  </Menu>
);

export default Navbar;
