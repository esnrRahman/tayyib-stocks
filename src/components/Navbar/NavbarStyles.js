import styled from 'styled-components';
import { Link } from 'react-router-dom';

import STYLE_CONSTANTS from '../../constants/StyleConstants';

export const StyledNavbarMenu = styled.div`
  border: none;
  box-shadow: none;
  display: flex;
  justify-content: space-evenly;
  background: ${STYLE_CONSTANTS.BLACK};
`;

export const StyledNavbarMenuItem = styled(Link)`
  &&&& {
    font-size: 24px;
    font-family: ${STYLE_CONSTANTS.TEXT_FONT_FAMILY};
    padding: 30px;
    color: ${STYLE_CONSTANTS.WHITE};
  }
`;
