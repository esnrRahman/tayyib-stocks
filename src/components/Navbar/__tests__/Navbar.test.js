import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

import Navbar from '../Navbar';

describe('<Navbar />', () => {
  it('renders correctly', () => {
    const wrapper = shallow(<Navbar />);
    expect(toJson(wrapper)).toMatchSnapshot();
  });
});
