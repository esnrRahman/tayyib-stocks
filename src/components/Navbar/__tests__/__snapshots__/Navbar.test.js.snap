// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Navbar /> renders correctly 1`] = `
<Menu
  fluid={true}
  widths={5}
>
  <MenuItem
    name="Home"
  />
  <MenuItem
    name="About Us"
  />
  <MenuItem
    name="Shariah Compliance"
  />
  <MenuItem
    name="Helpful Websites"
  />
  <MenuItem
    name="Contact Us"
  />
</Menu>
`;
